modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -3,7 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Secure properties can encrypt properties and store the encrypted text in secure properties files.
+Create secure configuration properties by encrypting them. To set up the encryption process:
+
+. Obtain the Mule Secure Configuration Property Extension module from Exchange.
+. Configure a secure properties file with this module, as described in <<Adding the Secure Configuration Properties Module>>. This file must point to or include the decryption key.
+. Define secure properties in any properties configuration file. Enclose the secured properties between the sequence `![value]`, as described in <<Configuration Files Content>>. The same properties configuration file can include secured and non-secured properties.
 
 == Adding the Secure Configuration Properties Module
```

Comment:
On the line above this section, add a line that contains an anchor, such as [[add_module]]. Then instead of using "<<Adding the Secure Configuration Properties Module>>", you can use <<add_module>> in the step.

Reply:
Changed.